test/components/DayPickerRangeController_spec.jsx

```diff
@@ -838,6 +838,32 @@ describe('DayPickerRangeController', () => {
             expect(isBlockedStub.callCount).to.equal(numVisibleDays);
           });
 
+          it('calls isBlocked for every visible day when startDate changes', () => {
```

Comment:
Note: I only added test cases to verify the call to the modifier function, because the code path followed when `startDate` / `endDate` change is otherwise identical to when `focusedInput` changes. I felt duplicating all the tests, with only the prop changing, would be low reward.